docs/docs/webchat/webchat-style.md

```diff
@@ -424,3 +423,45 @@ All these changes can be tested using the `botonic serve` command (`npm run star
 ![](https://botonic-doc-static.netlify.com/images/webchat_example.png)
 
 </details>
+
+## Style Compliance
+
+To keep the code nice and clean, and to avoid style conflicts, encapsulation is recommmended: the markup structure, style, and behavior can be hidden and separated from the rest of the code.
+
+To do so, you only need a slight change in your `webpack.config.js` file and replace the `stylesLoaderConfig` by the following one:
+
+```javascript
+const stylesLoaderConfig = {
+  test: /\.(scss|css)$/,
+  use: [{loader: 'style-loader', options: {
+    insert: function(element) {
+      if(!window._botonicInsertStyles)
+        window._botonicInsertStyles = []
+      window._botonicInsertStyles.push(element)
+    }
+  }}, 'css-loader', 'sass-loader'],
+}
+```
```

Comment:
The way this is explained here it seems that it's a separate thing from the `shadowDOM` option, but in reality it's not. If you enable `shadowDOM`, this piece of code needs to be in your webpack config so that styles are loaded in your shadowed DOM.

In any case, this piece of code is already added in our templates, so it's not something the user will have to add manually. So maybe it's better to leave it out of the docs to avoid confusion

Reply:
Section removed.